Configuration/Geometry/scripts/dict2023Geometry.py

```diff
@@ -353,6 +353,78 @@
             'from Geometry.EcalMapping.EcalMappingRecord_cfi import *',
         ],
         "era" : "run2_HE_2017, run2_HF_2017, run2_HCAL_2017, run3_HB, phase2_hcal, phase2_hgcal",
+    },
+    "C3" : {

```

Comment:
At this moment it is a bit premature to add a new or modify existing workflow. I shall update the README - but I shall do that in a separate PR (this one is signed by all and updating this PR will require it go through the entire cycle all over again)